doc/usage/configuration.rst

```diff
@@ -3655,6 +3656,10 @@ and which failures and redirects it ignores.
    It can be useful to detect unexpected redirects when using
    :option:`the fail-on-warnings mode <sphinx-build --fail-on-warning>`.
 
+   To deny all redirects, configure an empty dictionary (the default).
+
+   To follow all redirections, configure a value of :code-py:`None`.
```

Comment:
Given recent bugreport #13462 about unconfigured `linkcheck_allow_redirects` reported as unexpectedly `None` during `sphinx-build -b html ...` builds: we should check that running `sphinx-build -b linkcheck ...` for an affected project yields the expected `{}` default.